### PR TITLE
Mediator interface for EAR replication

### PR DESCRIPTION
**Pull request by ELENAGER**

### CODE REVIEW

File: controllers/array_action/array_mediator_ds8k.py [hunk 1]
```diff
@@ -685,18 +685,33 @@ def get_flashcopy_state(self, flashcopy_id):
     def get_replication(self, volume_internal_id, other_volume_internal_id, other_system_id):
         raise NotImplementedError
 
+    def get_ear_replication(self, volume_internal_id):
```
Comment (ArbelNathan) [inline controllers/array_action/array_mediator_ds8k.py:688]: I dont believe ds8k will ever have EAR

File: controllers/array_action/array_mediator_interface.py [hunk 2]
```diff
@@ -420,6 +420,24 @@ def get_replication(self, volume_internal_id, other_volume_internal_id, other_sy
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def get_ear_replication(self, volume_internal_id):
```
Comment (ArbelNathan) [inline controllers/array_action/array_mediator_interface.py:424]: this is not common or general for all storages

File: controllers/array_action/array_mediator_xiv.py [hunk 3]
```diff
@@ -517,18 +517,33 @@ def get_array_fc_wwns(self, host_name):
     def get_replication(self, volume_internal_id, other_volume_internal_id, other_system_id):
         raise NotImplementedError
 
+    def get_ear_replication(self, volume_internal_id):
```
Comment (ArbelNathan) [inline controllers/array_action/array_mediator_xiv.py:520]: [same](https://github.com/IBM/ibm-block-csi-driver/pull/582/files#r1009100546)

File: controllers/array_action/array_mediator_ds8k.py (hunk 1 above)
Comment (ELENAGER) [reply controllers/array_action/array_mediator_ds8k.py:688]: But it is raising not implemented error

Comment (ArbelNathan) [reply controllers/array_action/array_mediator_ds8k.py:688]: suggesting it will be implemented in the future..

**Pull request closed without merging**